pkg/storage/replica_command.go

```diff
@@ -53,6 +53,65 @@ import (
 
 var errTransactionUnsupported = errors.New("not supported within a transaction")
 
+// CommandArgs contains all the arguments to a command.
+// TODO(bdarnell): consider merging with storagebase.FilterArgs (which
```

Comment:
I've gotten a bit wary of TODOs which lack conviction. Let's either leave this out (and file an issue to look into it where we can accumulate pros and cons of the suggested action), or rephrase it as an intention if you're really planning to do it. Otherwise if you don't get around to it, someone else is going to be wondering what subtleties you had in mind for not recommending more definitively it be done.